### ISSUE TITLE
AWX Page generation takes a significant amount of time.

### ISSUE THREAD

**Issue by nevotheless** [ansible/awx#3726]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY

Ahoi! I've come to encounter a really annoying performance problem when loading any AWX Page. The awx_web container reports page generating times of couple of seconds. 

```bash
awx_web      | [pid: 129|app: 0|req: 13/63] 172.18.0.1 () {34 vars in 448 bytes} [Wed Apr 17 09:19:13 2019] GET /api/ => generated 173 bytes in 3636 msecs (HTTP/1.1 200) 9 headers in 328 bytes (1 switches on core 0)
```

##### ENVIRONMENT
* AWX version: 4.0.0
* AWX install method: docker on linux
* Ansible version:  2.5.1
* Operating System: Ubuntu 18.04
* Web Browser: Google Chrome 73.0.3683.103

##### STEPS TO REPRODUCE

If only i knew... All i have is a simple Ubuntu 18.04 Machine, which is behind a corporate proxy.
I already set `no_proxy` in the inventory file accordingly: `localhost, 127.0.0.1/32, *.domain, reverseproxy.domain`. 

##### EXPECTED RESULTS

Expected results were tested with my private macbook where i installed AWX the same way as i did with the work server, (besides proxy related stuff). 

I expect Page load times in this range.

```bash
awx_web      | [pid: 129|app: 0|req: 32/181] 172.18.0.1 () {46 vars in 1101 bytes} [Wed Apr 17 09:19:13 2019] GET /api/ => generated 173 bytes in 377 msecs (HTTP/1.1 200) 9 headers in 328 bytes (1 switches on core 0)
```
##### ACTUAL RESULTS

```bash
awx_web      | [pid: 140|app: 0|req: 13/63] 172.18.0.1 () {34 vars in 448 bytes} [Wed Apr 17 09:19:13 2019] GET /api/ => generated 173 bytes in 3636 msecs (HTTP/1.1 200) 9 headers in 328 bytes (1 switches on core 0)
``` 

I have this behaviour on every page sooner or later. Sometimes pages loading rather quickly but i assume it's because of caching mechanisms.

##### ADDITIONAL INFORMATION

I'm behind a corporate proxy. Which made me do weird things in the past, but with proper configuration and nice applications that respects env variables i got everything to work.

Thats why i appreciate awx inventory options for setting proxy server stuff, which works quite nicely so far.

**Comment by lijok** [comment]
When did you install each of the installations?
There was an issue that was patched yesterday #3719 
If you installed before then, try to reinstall

**Comment by nevotheless** [comment]
Thanks for the answer @lijok.

I managed to figure it out. It was related to a misconfigured reverse proxy server which didn't properly delegate web socket requests to the awx_web container.
